badparking/core/models.py

```diff
@@ -11,3 +13,23 @@ def __unicode__(self):
     class Meta:
         verbose_name = "Тип порушення"
         verbose_name_plural = "Типи порушень"
+
+
+class Claim(models.Model):
+    license_plates = models.CharField("Номери автомобілів", max_length=50, blank=True)
+    longitude = models.CharField("Довгота", max_length=50, blank=True)
+    latitude = models.CharField("Широта", max_length=50, blank=True)
+    city = models.CharField("Місто", max_length=30, blank=True)
+    address = models.CharField("Адреса", max_length=150, blank=True)
+    created_at = models.DateTimeField(editable=False, auto_now_add=True)
+
+    user = models.ForeignKey(User, verbose_name="Користувач")
+    crimetypes = models.ManyToManyField(CrimeType, verbose_name="Порушення")
+    images = models.ManyToManyField(MediaFileModel, blank=True, verbose_name="Зображення")
+
+    def __unicode__(self):
```

Comment:
We're using Python 3 so this should be `__str__` instead. Also it's purpose is to return something human-readable so maybe make it more descriptive if it's here?